src/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  width: 96px;
+  height: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #fff;
+} */
+
+body {
+  --basic-size: 16px;
+}
+
+.stars__row {
+  width: 96px;
+  display: flex;
+}
+
+.stars__star {
+  width: var(--basic-size);
+  height: var(--basic-size);
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
According to the checklist - background properties should be combined using a shorthand